### DISCUSSION TITLE
French translation

### DISCUSSION THREAD

**Discussion by caloup** [oqtane/oqtane.translations#37] [Show and tell]
I'm sorry but i don't know yet how to add translation file or pull request on github so i gave you a link for french version of the ressource files for oqtane.
I translated them because i found this project really interesting and i will try to learn a lot from this project.
Thanks for all the job you've done until now :-)

resx files : 
https://drive.google.com/file/d/1WgKnZX_Jx1FCabh3EDnhl3Hke8Duo2nZ/view?usp=sharing

compiled files : 
https://drive.google.com/file/d/1a52Jra-TmeB4obLBBDpwytxOXc78dB7c/view?usp=sharing

I used ressources from version 2 of octane

**Comment by hishamco** [comment]
Why you didn't push the translation here?

**Reply by leigh-pointer** [reply]
@hishamco If I read this correctly @caloup does not have the experience in using GitHub

**Reply by hishamco** [reply]
Oh I didn't see thanks

Thanks

**Reply by caloup** [reply]
Yes sorry ! I never used github before, but i will try to learn how to use it :)
